samples-aspnetcore-6x/resource-server/README.md

```diff
@@ -0,0 +1,64 @@
+# Okta ASP.NET Core Web API Resource Server Example
+
+This sample application authenticates requests against your Web API application, using access tokens.
+
+The access tokens are obtained via the [Implicit Flow][].  As such, you will need to use one of our front-end samples with this project.  It is the responsibility of the front-end to authenticate the user, then use the obtained access tokens to make requests to this resource server.
+
+## Prerequisites
+
+Before running this sample, you will need the following:
+
+* An Okta Developer Account, you can sign up for one at https://developer.okta.com/signup/.
+* An Okta Application, configured for Single-Page App (SPA) mode. This is done from the Okta Developer Console and you can find instructions [here][OIDC SPA Setup Instructions].  When following the wizard, use the default properties.  They are are designed to work with our sample applications.
+* One of our front-end sample applications to demonstrate the interaction with the resource server:
+  * [Okta Angular Sample Apps][]
+  * [Okta React Sample Apps][]
+  * [Okta Vue Sample Apps][]
+
+*Note:* This sample is pre-configured with an open CORS policy to make it easy to test with frontend projects. Make sure to restrict it to known origins in your production applications.
+
+A typical resource-server requires a frontend and a backend application, so you will need to start each process as described in the next section.
+
+## Running This Example
+
+### Backend
+
+Run the example with your preferred tool and write down the port of your Web API application to configure Okta afterwards.
+
+> **NOTE:** Although we highly recommend using HTTPS in production, we are using HTTP here to be compatible with our [SPA samples](#Prerequisites) that expect a protected endpoint on HTTP://localhost:7064/api/whoami. 
```

Comment:
I suggest we use the same ports as existing sample apps, so devs that are using our samples already don't have to update their configuration.